src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -187,22 +173,24 @@ public Base(final Request req) throws IOException {
         /**
          * Build a request for each part of the origin request.
          * @param req Origin request
+         * @param body Origin request body
          * @return The requests map that use the part name as a map key
          * @throws IOException If fails
          */
         private ConcurrentMap<String, List<Request>> buildRequests(
-            final Request req) throws IOException {
+            final Request req, final  ReadableByteChannel body)
+                throws IOException {
             final String header = new RqHeaders.Smart(
                 new RqHeaders.Base(req)
-            // @checkstyle MultipleStringLiteralsCheck (1 line)
             ).single("Content-Type");
             if (!header.toLowerCase(Locale.ENGLISH)
                 .startsWith("multipart/form-data")) {
                 throw new HttpException(
                     HttpURLConnection.HTTP_BAD_REQUEST,
                     String.format(
-                        // @checkstyle LineLength (1 line)
-                        "RqMultipart.Base can only parse multipart/form-data, while Content-Type specifies a different type: \"%s\"",
+                        "RqMultipart.Base can only parse ",
```

Comment:
@2686747 Don't change this, it will cause the message to be broken. Leave the suppression in place.